src/Form.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+class Form extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchBarValue: ''
+    }
+  }
+
+  render() {
+    return (
+      <form>
+        <input type='text' placeholder='Search Movie...'/>
```

Comment:
I'll try and make this a controlled form as I move into Iteration 2 of the spec.